Mathlib/Algebra/BigOperators/Ring.lean

```diff
@@ -22,6 +22,14 @@ variable {ι ι' α β γ : Type*} {κ : ι → Type*} {s s₁ s₂ : Finset ι}
 
 namespace Finset
 
+theorem sum_tsub_distrib [AddCommMonoid α] [PartialOrder α] [ExistsAddOfLE α]
```

Comment:
This apparently can be placed in `Mathlib.Algebra.BigOperators.Group.Finset`, which seems more appropriate.

Can you add the multiset version too?